homeassistant/components/recorder/models.py

```diff
@@ -224,8 +224,7 @@ class Statistics(Base):  # type: ignore
     __tablename__ = TABLE_STATISTICS
     id = Column(Integer, primary_key=True)
     created = Column(DATETIME_TYPE, default=dt_util.utcnow)
-    source = Column(String(32))
-    statistic_id = Column(String(255))
+    metadata_id = Column(Integer)
```

Comment:
Relations?

```suggestion
    metadata_id = Column(
        Integer, ForeignKey(f"{TABLE_STATISTICS_META}.id", ondelete="CASCADE"), index=True
    )
```

Should this be indexed?